### PR TITLE
[WIP] Update to use new Daemon base class in ocean

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**
See https://github.com/sociomantic-tsunami/ocean/pull/544.

### CODE REVIEW

File: src/dhtnode/main.d [hunk 1]
```diff
@@ -329,18 +324,21 @@ public class DhtNodeServer : DaemonApp
 
     ***************************************************************************/
 
-    public override void onSignal ( int signum )
+    protected override void onSignal ( int[] signals )
```
[the remark this answers is not in the document]
Comment (mihails-strasuns) [reply src/dhtnode/main.d]: I would also recommend sorting signals by "severity" before passing to this handler.

[the remark this answers is not in the document]
Comment (gavin-norman-sociomantic) [reply src/dhtnode/main.d]: Mm, interesting idea. Is there a standard severity sort order?

[the remark this answers is not in the document]
Comment (mihails-strasuns) [reply src/dhtnode/main.d]: Not directly. But all signals can be split into groups by default action - core dump, regular termination, ignoring. That directly matches severity IMHO.

**Comment by matthias-wende-sociomantic** [comment]
This is not being worked on for over a year. I'll close it until some actual development is continued.

**Pull request closed without merging**